src/components/DocumentationTopic/Hero/HierarchyCollapsedItems.vue

```diff
@@ -20,8 +19,11 @@
       <span class="indicator"><EllipsisIcon class="icon-inline toggle-icon" /></span>
     </button>
     <ul class="dropdown" ref="dropdown" :class="{ collapsed }">
-      <li v-for="topic in topicsWithUrls" class="dropdown-item" :key="topic.title">
-        <router-link class="nav-menu-link" :to="topic.url">{{ topic.title }}</router-link>
+      <li v-for="topic in formattedTopics" class="dropdown-item" :key="topic.title">
+        <router-link v-if="topic.url" class="nav-menu-link" :to="topic.url">
+          {{ topic.title }}
+        </router-link>
+        <span class="nav-menu-link" v-else>{{ topic.title }}</span>
```

Comment:
Should we give this a different class name? I _think_ we use the `nav-menu-link` class for link related colors, which this `span` shouldn't get.

Reply:
I ended up refactoring this component using `NavMenuLink` instead. https://github.com/apple/swift-docc-render/pull/772/commits/34d4fdd3de1c5a0b70df762dbee60b9ad9b6f4f4 which includes aria-labels too